lab_1/main.py

```diff
@@ -51,8 +94,18 @@ def create_language_profile(language: str, text: str, stop_words: list) -> dict
     :param stop_words: a list of stop words
     :return: a dictionary with three keys – name, freq, n_words
     """
-    pass
 
+    if not isinstance(language, str):
+        return None
+    if not isinstance(text, str):
+        return None
+    if not isinstance(stop_words, list):
+        return None
+    tokens = tokenize(text)
+    tokens = remove_stop_words(tokens, stop_words)
+    freq_dict = calculate_frequencies(tokens)
```

Comment:
can join all transformations here
or at least tokenize and remove stop words

Reply:
done